README.md

```diff
@@ -6,429 +6,10 @@
-Manipulating Chef Resources portion of this document.
-
-Unlike a role, node, client, or environment, a data bag is a container for other resources. These other resources are Data Bag Items. Data Bag Items behave slightly different than other resources. Data Bag Items can have an arbitrary attribute hash filled with any key values that you would like. The one exception is that every Data Bag Item __requires__ an 'id' key and value. This identifier is the name of the Data Bag Item.
-
-### Creating a Data Bag Item
-
-```ruby
-ridley   = Ridley.new(...)
-data_bag = ridley.data_bag.create(name: "ridley-test")
-
-data_bag.item.create(id: "appconfig", host: "reset.local", user: "jamie") #=> #<Ridley::DataBagItemObject: chef_id:appconfig, host="reset.local", user="jamie">
-```
-
-Environment Resource
---------------------
-
-### Setting Attributes
-
-Setting a default environment attribute is just like setting a node level default attribute
-
-```ruby
-ridley = Ridley.new(...)
-production_env = ridley.environment.find("production")
-production_env.set_default_attribute("my_app.proxy.enabled", false)
-production_env.save #=> true
-```
-
-And the same goes for setting an environment level override attribute
-
-```ruby
-production_env.set_override_attribute("my_app.proxy.enabled", false)
-production_env.save #=> true
-```
-
-Role Resource
--------------
-
-### Role Attributes
-
-Setting role attributes is just like setting node and environment attributes
-
-```ruby
-ridley = Ridley.new(...)
-my_app_role = ridley.role.find("my_app")
-my_app_role.set_default_attribute("my_app.proxy.enabled", false)
-my_app_role.save #=> true
-```
-
-And the same goes for setting an environment level override attribute
-
-```ruby
-my_app_role.set_override_attribute("my_app.proxy.enabled", false)
-my_app_role.save #=> true
-```
-
-Sandbox Resource
-----------------
-
-Search Resource
----------------
-
-```ruby
-ridley = Ridley.new(...)
-ridley.search(:node)
-ridley.search(:node, "name:ridley-test.local")
-```
-
-Search will return an array of the appropriate Chef Objects if one of the default indices is specified. The indices are
-
--  node
--  role
--  client
--  environment
-
-User Resource
--------------
-
-### Regenerating a user's private key
-
-Works the same way as with a client resource.
-
-### Authenticating a user's password
-
-```ruby
-ridley = Ridley.new(...)
-ridley.user.authenticate('username', 'password')
-ridley.user.find('username').authenticate('password')
-```
-
-Authors and Contributors
-------------------------
-- Jamie Winsor (<jamie@vialstudios.com>)
-- Kyle Allan (<kallan@riotgames.com>)
-
-Thank you to all of our [Contributors](https://github.com/reset/ridley/graphs/contributors), testers, and users.
+This is the HTTP Client API for Berkshelf.  It is supported for that purpose, but for all external purposes its use
+is deprecated.  Chef users should use the `Chef::ServerAPI` class in the `chef` gem.
```

Comment:
Should we mention the `chef-api` gem? Think it's worth presenting an alternative to dealing with the entire chef stack.